router/record.py

```diff
@@ -19,7 +20,7 @@
 
 
 @error_handler
-@router.get("/detail/{sport_id}")  # , response_model=RecordBody)
+@router.get("/detail/{sport_id}", dependencies=[Depends(check_token)])  # , response_model=RecordBody)
```

Comment:
Too duplicate `Depends(check_token)` use dependencies=Depends(check_token) in APIRouter instead

Reply:
you can look docs here https://fastapi.tiangolo.com/reference/apirouter/

Reply:
uncomment response_model for me pls too!